transports/sftp/src/test/java/org/mule/transport/sftp/SftpReceiverRequesterUtilTestCase.java

```diff
@@ -58,4 +75,138 @@ public void clientIsReleasedWhenRetrievingFails() throws Exception
             verify(connector).releaseClient(endpoint, sftpClient);
         }
     }
+
+    /**
+     * Verifies an stable file (i.e. a file whose size doesn't change) is returned when fileAge is disabled.
+     */
+    @Test
+    public void testReturnsStableFile() throws Exception
+    {
+        final String stableFile = "file1";
+        when(connector.getFileAge()).thenReturn(-1L);
+        when(sftpClient.listFiles()).thenReturn(new String[] {stableFile});
+        when(sftpClient.getSize(stableFile)).thenReturn(1024L);
+        SftpReceiverRequesterUtil requesterUtil = new TestSftpReceiverRequesterUtil(endpoint);
```

Comment:
Add extra lines to separate the different parts of the tests to make them more readeable (setup, execute, assert)